index.html

```diff
@@ -1676,16 +1676,10 @@ <h2>Additional
 <section id="4Concepts.Format">
 <h2>PNG datastream</h2>
 
-<!-- Maintain a fragment named "4Concepts.FormatChunks" to preserve incoming links to it -->
```

Comment:
Don't delete that section and id (or was it moved, and the diff didn't find it?)

Reply:
Will revert. We can fix later.